test/functional/mempool_accept_wtxid.py

```diff
@@ -104,13 +101,12 @@ def run_test(self):
             "allowed": False,
             "reject-reason": "txn-already-in-mempool"
         }])
-        testres_child_two = node.testmempoolaccept([child_two.serialize().hex()])[0]
-        assert_equal(testres_child_two, {
+        assert_equal(node.testmempoolaccept([child_two.serialize().hex()]), [{
```

Comment:
Keeping the index would reduce the diff
```suggestion
        assert_equal(node.testmempoolaccept([child_two.serialize().hex()])[0], {
```

Reply:
Done!